openshift_cli_installer/libs/clusters/ipi_cluster.py

```diff
@@ -51,6 +52,7 @@ def _prepare_ipi_cluster(self):
             )
         )
         self.set_cluster_install_version()
+        self._set_docker_config_file()
```

Comment:
this is needed only for `_ipi_download_installer` right?, if so, should be called there